src/cayenne/conf.clj

```diff
@@ -166,7 +166,7 @@
   (set-param! [:upstream :unixref-url] "http://doi.crossref.org/search/doi?pid=cnproxy@crossref.org&format=unixref&doi=")
   (set-param! [:upstream :unixsd-url] "http://doi.crossref.org/search/doi?pid=cnproxy@crossref.org&format=unixsd&doi=")
   (set-param! [:upstream :prefix-info-url] "http://doi.crossref.org/getPrefixPublisher/?prefix=")
-  (set-param! [:upstream :ra-url] "http://hdl.handle.net/api/handles/")
+  (set-param! [:upstream :ra-url] "https://doi.crossref.org/doiRA/")
```

Comment:
Because this is an entirely different API, I might be tempted to use a different config name for deployment clarity. Like `:doi-ra-url`. Thoughts?

Reply:
Yeah, good idea. I can change that now if you like? As far as I can see it's only used in one place.

Reply:
Yes please. If it's mentioned in a config file (it shouldn't be) it'll need changing in any case.